docs/apireference.rst

```diff
@@ -0,0 +1,105 @@
+==================================
+API Reference
+==================================
+
+
+Surfaces
+--------
+
+.. autosummary::
```

Comment:
> The main reason I suggested this is because I found there to be too many links to click before I got to the API. And the API is all exposed in `xtgeo.`, while this shows for example `xtgeo.surface.RegularSurface` instead of `xtgeo.RegularSurface`

Excellent! I change the approach and use your suggestion.